docker/Dockerfile

```diff
@@ -2,8 +2,11 @@
 FROM registry.access.redhat.com/rhel7:7.8-265
 
 RUN set -x && \
+    yum-config-manager --save --setopt=skip_missing_names_on_install=False && \
+    yum-config-manager --save --setopt=skip_missing_names_on_update=False && \
```

Comment:
can those `--setopt` arguments be part of `yum install` command instead of using `yum-config-manager` to persist and then remove/undo?

Reply:
fixed this @cvarjao. Thanks